src/ipc/ipc4/helper.c

```diff
@@ -201,7 +201,7 @@ static struct comp_buffer *ipc4_create_buffer(struct comp_dev *src, struct comp_
 	src_cfg = (struct ipc4_base_module_cfg *)comp_get_drvdata(src);
 	sink_cfg = (struct ipc4_base_module_cfg *)comp_get_drvdata(sink);
 
-	buf_size = MAX(src_cfg->obs, sink_cfg->ibs);
+	buf_size = src_cfg->obs * 2;
```

Comment:
Can we also have a comment here explaining the logic.

Reply:
sure